config/config.go

```diff
@@ -853,6 +854,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("lmt.enforce", true)
 	v.SetDefault("currency_converter.fetch_url", "https://cdn.jsdelivr.net/gh/prebid/currency-file@1/latest.json")
 	v.SetDefault("currency_converter.fetch_interval_seconds", 1800) // fetch currency rates every 30 minutes
+	v.SetDefault("currency_converter.stale_rates_seconds", 7200)    // fetched currency rates are stale at 120 minutes
```

Comment:
I think we should default this to `0` and consider `rates` to be never stale if it is set to `0` i.e. the behavior that we have today. So, if somebody who is hosting prebid server today don't use this new config they will still see the same behavior as before.